packages/dsi-logo-maker/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+
+- Updated `fillAnimatedCanvas` to show the new `debounceInputs` setting
```

Comment:
If we plan keeping this change, then I would also add an example for `debounceDelay`.